src/VirtualClient/VirtualClient.Contracts/VirtualClientComponentExtensions.cs

```diff
@@ -20,6 +19,64 @@ namespace VirtualClient.Contracts
     /// </summary>
     public static class VirtualClientComponentExtensions
     {
+        // Example Format:
+        // fio_{IOType}_{BlockSize}_{FileSize}_thmax{MaxThreads}
+        private static readonly Regex ParameterReferenceExpression = new Regex(@"(\x7B[\x20-\x7A\x7C\x7D-\x7E]+\x7D)|(\x5B[\x20-\x5A\x5C\x5E-\x7E]+\x5D)", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
Could these be translated into literal characters? It will be more readable

Reply:
They can be but the Regex was a bit more flaky than I would like. This is why I went with the ASCII character references. The example above is meant to shed light on what an actual string looks like for the sake of readability.